strawberry_django/fields/types.py

```diff
@@ -103,6 +103,86 @@ class ManyToManyInput:
         }
     )
 
+try:
+    from django.contrib.gis import geos
+    from django.contrib.gis.db import models as geos_fields
+
+    Point = strawberry.scalar(
+        NewType("Point", Tuple[float, float, Optional[float]]),
+        serialize=lambda v: v.tuple if isinstance(v, geos.Point) else v,
+        parse_value=lambda v: geos.Point(v),
+        description="Represents a point as `(x, y, z)` or `(x, y)`.",
+    )
+
+    LineString = strawberry.scalar(
+        NewType("LineString", Tuple[Point]),
+        serialize=lambda v: v.tuple if isinstance(v, geos.LineString) else v,
+        parse_value=lambda v: geos.LineString(v),
+        description="""
+            A geographical line that gets multiple 'x, y' or 'x, y, z' tuples
+            to form a line.
+        """,
```

Comment:
This is a literal string, so you are really adding the whitespaces in the string. Probably format this as:

```python
description = (
    "Some long string "
    "split into multiple lines"
)
```

And ditto below

Reply:
Done 1bf5721f286d3b07a49b98f32728cd1b2c15de68